core/core.mk

```diff
@@ -90,9 +90,21 @@ base-prefix :=
 
 CFG_CRYPTOLIB_NAME ?= tomcrypt
 CFG_CRYPTOLIB_DIR ?= core/lib/libtomcrypt
+
+ifeq ($(CFG_CRYPTOLIB_NAME), tomcrypt)
+CFG_CRYPTO_TOMCRYPT := y
+else ifeq ($(CFG_CRYPTOLIB_NAME), mbedtls)
+CFG_CRYPTO_MBEDTLS := y
+else
+$(error Error: $(WITH_CRYPTO) is not supported.)
```

Comment:
The purpose of CFG_CRYPTOLIB_NAME is to allow for platform specific crypto libs. This logic does not allow that.

Reply:
@deebee-v2 thanks for your comments.
Yes, it should not add this `error` message, I will delete it.